src/components/molecules/UserProfilePicture/UserReactions.scss

```diff
@@ -0,0 +1,115 @@
+@import "scss/constants";
+
+// TODO: should we move these to using CSS --variables rather than these SCSS vars
+$reactionSize: max(2.5vh, 2vw);
```

Comment:
I think that we probably can get `$reactionSize`, based on `$avatarSize` since they are used in tandem. Wdyt?

Reply:
I actually had the code that way originally, but then sort of changed my mind on it to remove the 'dependency' on avatar size. I think we could pull this (and likely avatar size) back out into our main constants, at which point it would make sense to me to use that way. Then we can 'configure them all in the same place'.

Reply:
Decided not to directly define them in terms of one another, but refactored both into `scss/constants` in https://github.com/sparkletown/sparkle/pull/1323/commits/3c6adcbfde6c7fc244a209afeac7de7460f79a05